chia/types/eligible_coin_spends.py

```diff
@@ -23,7 +23,8 @@
 class EligibilityAndAdditions:
     is_eligible_for_dedup: bool
     spend_additions: list[Coin]
-    is_eligible_for_ff: bool
+    # This is None if the spend is not eligible for fast forward
```

Comment:
"otherwise it is..."?
It's the puzzle hash of the coin being spent, right?
I think it would also be good to say that this being set means this spend is eligible for fast forward and that we use this to check if the singleton has an unspent coin or not

Reply:
Yep, I elaborated accordingly in the comment.